### ISSUE TITLE
Push notifications from multiple providers to multiple devices

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ntrees/javapns#62]
```
I'm using javapns-jdk16-169.

I would like to send multiple messages to multiple devices at once, using two 
threads, how I can do?

```

Original issue reported on code.google.com by `alyr...@gmail.com` on 9 Aug 2011 at 4:26

**Comment by GoogleCodeExporter** [comment]
```
version 1.7 (in svn) should solve this for you
```

Original comment by `idbill.p...@gmail.com` on 30 Aug 2011 at 7:57

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.0, closing.
```

Original comment by `sype...@gmail.com` on 7 Sep 2011 at 4:32
- Changed state: **Fixed**
